packages/editor-ui/src/components/MarkdownViewer.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+	<div ref="editor" :class="$style.markdown" v-html="getContent" />
+</template>
+
+<script lang="ts">
```

Comment:
why is this not a component in the design system?

Reply:
Moved